docs/docs/reference/node-sdk.md

```diff
@@ -150,3 +150,26 @@ The return value looks like this:
     ];
 }
 ```
+
+## Getting a list of all Connections for a specific connectionId
```

Comment:
This is a great addition. Do you mind also updating the [docs](https://docs.nango.dev/reference/connections-api#getting-a-list-of-all-connections) of the HTTP API please?